scripts/tests/repro-yaml-config.sh

```
@@ -0,0 +1,166 @@
+
+cleanup() {
+  printf '  cleaning up test schema %s…\n' "$SCHEMA"
+  bun --cwd "$REPO_ROOT" -e "
+    const postgres = (await import('postgres')).default;
+    const port = Number.parseInt(process.env.GENIE_PG_PORT || '19642', 10);
+    const sql = postgres({ host: '127.0.0.1', port, database: 'postgres', user: 'postgres' });
+    try { await sql\`DROP SCHEMA IF EXISTS \${sql(process.env.TEST_SCHEMA)} CASCADE\`; }
+    finally { await sql.end({ timeout: 1 }); }
+  " >/dev/null 2>&1 || true
+  rm -rf "$WORK"
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)